Tests/UI/New UI/BillingForm/Validator/AddressLine2ValidatorTests.swift

```diff
@@ -0,0 +1,19 @@
+import XCTest
+@testable import Frames
+
+class AddressLine2ValidatorTests: XCTestCase {
+
+    func testValidationWhenTextIsEmptyThenShouldBeTrue() {
+        let expectedType = BillingFormCellType.addressLine2
+        let text = ""
+        let isValid = expectedType.validator.validate(text: text)
+        XCTAssertEqual(isValid, true)
+    }
+    
+    func testValidationWhenTextIsNonEmptyThenShouldBeFalse() {
+        let expectedType = BillingFormCellType.addressLine2
+        let text = "addressLine2"
+        let isValid = expectedType.validator.validate(text: text)
+        XCTAssertEqual(isValid, false)
```

Comment:
When asserting against a boolean it may be clearer to use the correct assertion API (ie: `XCTAssertTrue`, `XCTAssertFalse`). I see this was missed across on many occasions so addressing it may provide _muscle memory_.